barretenberg/cpp/src/barretenberg/relations/poseidon2_external_relation.hpp

```diff
@@ -0,0 +1,122 @@
+#pragma once
```

Comment:
These are in master now so you should merge master or rebase onto master or something to make sure you're not overwriting  changes that were made during review.

Reply:
fixed